### DISCUSSION TITLE
Oi Grandad compatibility on Mac M1 chip?

### DISCUSSION THREAD

**Discussion by copier94** [publicsamples/Oi-Grandad#18] [Q&A] [answered]
Hi, I have successfully downloaded Oi Grandad on my old Mac (Intel chip) and used it on Logic Pro X. I recently bought  a new Macbook Pro with an M1 Pro chip and see Oi Grandad when I open my Plug-in Manager on LPX and have gotten it validated, however, I can't find it anywhere on Logic. Any info or advice would be much appreciated. Thanks!

*Also, I'm a little confused in the first place because the install instructions for Mac say there should be a .dmg file when you unzip the download but I couldn't find one. Thanks

**Comment by RickTaylor79** [comment, marked as answer]
Hmm, that's an odd one. It's definitely compatible with M1 as that's what I've used to build it. You could try doing a "Full AudioUnit reset" from the Plugin manager. 

Alternatively, this forum post seems to match what you are experiencing:

https://www.logicprohelp.com/forums/topic/134057-validated-plugins-not-appearing-for-selection-on-my-tracks-solved/

"None of my Waves plugins were showing after validation, in addition to several other workhorse plugins. This was scary, but it worked for me:

1. Delete these:

~/Library/Preferences/com.apple.audio.AUHostingService.x86-64.plist 
~/Library/Preferences/com.apple.audio.InfoHelper
~/Library/Caches/AudioUnitCache
~/Library/Caches/com.apple.audio.AUHostingService.x86-64
~/Library/Caches/com.apple.audio.InfoHelper
2. Restart Logic

Everything is currently appearing and will hopefully stay that way"

Yes, that's becuase I'm an idiot and forgot to upload it last night. Here's the DMG:
https://github.com/publicsamples/Oi-Grandad/releases/download/0.99/OiGrandad.0.99.dmg

**Reply by copier94** [reply]
Excellent! It worked. What I did was your suggestion to do a "Full Audio Unit Reset". Then downloaded the .dmg files. Thanks!

**Reply by RickTaylor79** [reply]
Ah, glad that worked. Much easier than trashing all those preferences.
